public.yaml

```diff
@@ -164,6 +164,21 @@ spec:
       readOnly: True
     - mountPath: /etc/salt/pki/minion
       name: salt-minion-pki
+  - name: salt-minion-admin
+    image: sles12/salt-minion:2016.11.4
+    volumeMounts:
+    - mountPath: /etc/pki
+      name: salt-minion-admin-certificates
+    - mountPath: /etc/salt/minion.d/minion.conf
+      name: salt-minion-admin-config
+      readOnly: True
+    - mountPath: /etc/salt/minion.d/signing_policies.conf
+      name: salt-minion-admin-signing-policies
+      readOnly: True
+    - mountPath: /etc/salt/grains
+      name: salt-minion-admin-grains
+    - mountPath: /etc/salt/pki/minion
+      name: salt-minion-pki
```

Comment:
This will clash with the `ca` container (that has the same mountpoint on the host), so they will override each other generated certificates when registering against the `salt-master`.